.env.test

```diff
@@ -6,3 +6,5 @@
 # Reference: https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
 
 DATABASE_URL=postgres://postgres@localhost:5432/roda_test
+AUTH0_CLIENT_ID=123456
```

Comment:
Is it worth stubbing `AUTH0_CLIENT_ID` and `AUTH0_DOMAIN` in the tests that require them rather than setting them globally here?